### DISCUSSION TITLE
Deepfake技術

### DISCUSSION THREAD

**Discussion by 109302003** [1101-GS4719-C/Python-Community-GS4719-C#101] [General]
最近的頭條新聞Deepfake 是深度學習（Deep Learning）與偽造（Fake）。深偽可以應用於製造出逼真的假影片。如果把此技術用於影像合成領域，可使電影或遊戲製作更為省時；但在現實中，可能會觸及法律問題。

[https://www.bnext.com.tw/article/57308/deepfake-autoencoder-gan]()

**Comment by chia-kai-chang** [comment]
也可以閱讀一下鏡週刊的報導：https://www.mirrormedia.mg/projects/deepfaketaiwan/

昨天的公視有話好說也有討論這個議題：https://youtu.be/IV1fgynA7QU
